apps/docs/src/app/core/component-docs/list/list-header/list-header.component.html

```diff
@@ -1,10 +1,43 @@
 <header>List</header>
-<description
-    >The List Component can be used to display a list of items with simple information such as scopes, names, etc.
-    <br /><br />
-    List item anchors will be keyboard navigable via the tab key provided they have <code>[routerLink]</code> or
-    <code>[attr.href]</code> attributes. List action buttons and checkboxes can be navigated via the tab key. List
-    checkboxes can be selected with the space bar.
+<description>
+    In SAP Fiori, we distinguish between tables and lists.
+    Both usually contain homogeneous data, but lists generally have rather basic data,
+    whereas the data in tables tends to be more complex. Lists are mostly used in the master list
+    for a master-detail scenario using the flexible column layout, as well as in popovers or dialogs.
+
+    The list component displays a list of items with basic homogeneous information.
+    Lists are usually used in the master list for a master-detail scenario using a flexible column layout,
```

Comment:
Looks like you are repeating the "Lists are usually used in the master list for a master-detail scenario using a flexible column layout" bit twice